reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -45,20 +45,20 @@ import java.io.File
  * for each project in [ScanRecord.analyzerResult] and an additional sheet that summarizes all dependencies.
  */
 class ExcelReporter : TableReporter() {
-    val DEFAULT_COLUMNS = 5
+    private val defaultColumn = 5
```

Comment:
This seems like a broken inspection, it should actually propose to use `const val DEFAULT_COLUMNS = 5` as in the style guide:
https://kotlinlang.org/docs/reference/coding-conventions.html#property-names

It seems to propose to use `const` for `FIELDS` below, weird.

Also it should be plural `defaultColumns`.

Reply:
Unfortunately, "Const 'val' are only allowed on top level or in objects".

> Also it should be plural `defaultColumns`.

Fixed that!